hiringapp/tasks.py

```diff
@@ -0,0 +1,59 @@
+import string
+from .mailutils import create_messages
+from django.contrib.auth.models import User
+from django.utils.crypto import get_random_string
+from .mailutils import get_mail_service,send_message
+from celery import shared_task
+from . import models
+from .utils import EmailType
+from django.utils import timezone
+from datetime import datetime,timedelta
+from .utils import ActivityStatus
+
+@shared_task
+def send_emails(id,email_type):
+    submission=models.Submission.objects.get(activity_uuid=id)
+    try:
+        message=create_messages(submission,email_type)
+        service=get_mail_service(submission.invitation_host)
+        sent = send_message(service,'me', message)
+        models.MailSummary.objects.create(mail_type=email_type,activity_uuid=id,candidate_name=submission.candidate_name,date_of_mail=timezone.now()) 
+        if email_type==EmailType.ActivityExpired.value or email_type==EmailType.ActivitySolution.value:
+            print('{} mail sent successfully to {} activity_uuid {}'.format(email_type,submission.invitation_host.get_username(),submission.activity_uuid))
+        else:            
+            print('{} mail sent successfully to {} activity_uuid {}'.format(email_type,submission.candidate_name,submission.activity_uuid))
+    except:
```

Comment:
never do full try-except blocks. It is a bad practice. Why? Because it tells me you don't know how your code will run.